src/Compilers/CSharp/Test/Emit/Attributes/InternalsVisibleToAndStrongNameTests.cs

```diff
@@ -1236,48 +1270,82 @@ public void SignModuleKeyFileBogus()
         {
             string s = @"[assembly: System.Reflection.AssemblyKeyFile(""bogus"")] public class C {}";
 
-            var other = CreateStandardCompilation(s, options: TestOptions.ReleaseModule.WithStrongNameProvider(s_defaultProvider));
+            var other = CreateStandardCompilation(s, options: TestOptions.ReleaseModule.WithStrongNameProvider(s_defaultDesktopProvider));
 
             //shouldn't have an error. The attribute's contents are checked when the module is added.
             var reference = other.EmitToImageReference();
 
             s = @"class D {}";
 
-            other = CreateStandardCompilation(s, new[] { reference }, TestOptions.ReleaseDll.WithStrongNameProvider(s_defaultProvider));
+            other = CreateStandardCompilation(s, new[] { reference }, TestOptions.ReleaseDll.WithStrongNameProvider(s_defaultDesktopProvider));
             other.VerifyDiagnostics(Diagnostic(ErrorCode.ERR_PublicKeyFileFailure).WithArguments("bogus", CodeAnalysisResources.FileNotFound));
         }
 
+        [Fact()]
+        public void AttemptToStrongnameWithOnlyPublicKey()
```

Comment:
`StrongName` in test name here and below.

Reply:
Done.